### PR TITLE
updated dp-design-system commit ref

### PR DESCRIPTION
**Pull request by IncredibleMrTim**
### What

This PR is to get the general page layout styling in place.  There are still outstanding items, but these will be tackled in subsequent PR's.  _Note that this feature is currently switched off in the production app._

fixes:
- filter checkbox and results separation and alignment
- alignment and padding of the filter title and sort dropdown
- Rearranged the DOM structure of the search template to allow for easier styling
- pointed to the config to the new dp-design-system commit
- set the `SearchDisabled` go flag to false.  this allows the use of the site-wide search in the search results page
- updated search results page styling to confirm with figma designs

### How to review

- view figma design: https://www.figma.com/file/kAyqXh8f02S0UXauLubSkF/Beta-release-reference?node-id=0%3A1
- the new layout, although not perfect, should be far more aligned with the design 

__Notes__:
- I am aware that the pagination at the bottom is currently not styled correctly

### Who can review

Anyone

### CODE REVIEW

File: assets/templates/search.tmpl [hunk 1]
```diff
@@ -1,24 +1,34 @@
...
+      </div>
+      <div class="search__filter__content col--border-top">
+        {{ template "partials/filter" . }}
+      </div>
+    </section>
+
+    <section
+      class="col col--md-34 col--lg-40 margin-left-md--1"
+      aria-label="Search results"
+    >
```
Comment (NoelleDL) [inline assets/templates/search.tmpl:23]: Could you move the chevrons up to the end of line? Both here and line 10.

File: assets/templates/partials/filter.tmpl [hunk 2]
```diff
@@ -8,7 +8,7 @@
     <div class="margin-bottom-md--2 flush-col js-mobile-filters">
         <div class="background--mercury border-top--iron-md padding-top-sm--2 padding-top-md--1 padding-right-sm--1 padding-right-md--1 padding-bottom-sm--2 padding-bottom-md--2 padding-left-sm--1 padding-left-md--1 js-mobile-filters__contents">
             <fieldset class="ons-fieldset">
-                <legend class="ons-fieldset__legend">
+                <legend class="ons-fieldset__legend ">
```
Comment (NoelleDL) [inline assets/templates/partials/filter.tmpl:11]: Was this extra space intentional?

Comment (IncredibleMrTim) [reply assets/templates/partials/filter.tmpl:11]: fixed

File: assets/templates/search.tmpl (hunk 1 above)
Comment (IncredibleMrTim) [reply assets/templates/search.tmpl:23]: fixed

File: assets/templates/search.tmpl [hunk 3]
```diff
@@ -1,24 +1,32 @@
...
+        class="col col--md-12 col--lg-18 padding-top-md--2 padding-right-md--1 padding-bottom-md--4 padding-left-sm--1 padding-left-md--1 js-mobile-filters__title">
+        <h3 class="font-size--18">
+          {{ localise "FilterResults" $lang 4 }}
+        </h3>
+      </div>
+      <div class="search__filter__content col--border-top">
+        {{ template "partials/filter" . }}
+      </div>
+    </section>
+
```
Comment (NoelleDL) [inline assets/templates/search.tmpl:18]: Please remove the empty line.

Comment (IncredibleMrTim) [reply assets/templates/search.tmpl]: fixed

**Review by NoelleDL — APPROVED** [review]

**Pull request merged**